odoo/addons/base/models/res_lang.py

```diff
@@ -155,7 +169,190 @@ def fix_datetime_format(format):
             tools.resetlocale()
 
     @api.model
-    def install_lang(self):
+    def _get_i18n_url(self, base, lang):
+        """ Generate the URL to fetch the translation resource
+            e.g. https://nightly.odoo.com/i18n/13.0/fr.tar.xz
+        """
+        version = release.version
+        return urljoin(urljoin(urljoin(base,
+                                       "i18n/"),
+                                       version),
+                                       lang + ".tar.xz")
```

Comment:
Seems unnecessary to use urljoin for the path we're providing, why not a regular format string or str.join? It's not like the separator can change or anything.

Reply:
@rim-odoo no, it's an URL